src/main/java/module-info.java

```diff
@@ -25,6 +25,11 @@
     exports tools.jackson.databind;
     exports tools.jackson.databind.annotation;
     exports tools.jackson.databind.cfg;
+    exports tools.jackson.databind.datetime;
```

Comment:
While this could work, it'd be nice to try to indicate more separation.
How about 

    tools.jackson.databind.ext.datetime

? (since we already have `ext.jdk8` for `Optional` etc)

Reply:
moved

Reply:
Thanks!